addon/components/copy-button.js

```diff
@@ -13,10 +13,16 @@ export default Ember.Component.extend({
     'clipboardTarget:data-clipboard-target',
     'clipboardAction:data-clipboard-action',
     'buttonType:type',
-    'disabled'
+    'disabled',
+    'aria-label'
   ],
 
   /**
+   * @property {String} aria-label - aria-label attribute for button element
+   */
+  'aria-label': '',
```

Comment:
I think the default can be removed. I added it as documentation but the API can be gleaned from the attributeBindings. Will rebase.